### ISSUE TITLE
Feature request: 3D preview for cubemaps

### ISSUE THREAD

**Issue by SaschaWillems** [baldurk/renderdoc#277]
Right now you can display separate faces of a cubemap in the texture viewer. Having a real 3D preview of the cubemap where you can look around would be a nice addition though, especially for dynamic cubemaps.

The PowerVR texture tool e.g. has such a feature that allows you to preview cubemaps in 3D before saving the texture. While that only helps you checking if you have selected the right files for your cubemap, in a tool like RenderDoc this could also be used to verify the address modes or your cube sampler, etc.

**Comment by baldurk** [comment]
Yes this has been on the [roadmap](https://github.com/baldurk/renderdoc/wiki/Roadmap) for a long long time.

There's a few things to do first though since this requires a few changes in how textures are rendered (since it needs to update with camera controls not just with pan). I have some of those on a side branch with improved mesh import/export using assimp but I need to tidy it up and merge it.

**Comment by SaschaWillems** [comment]
Ah sorry. Should have checked the road map first. Just thought that this would be a neat feature while debugging a dynamic cube mapping example :wink:

**Comment by Velro** [comment]
@baldurk you briefed me on how you would like this implemented back January, I never got around to it...  you good with me trying to attack it again?
